Bugzilla/Install/Filesystem.pm

```diff
@@ -405,8 +404,17 @@ sub FILESYSTEM {
         "skins/yui3.css"          => { perms     => CGI_READ,
                                        overwrite => 1,
                                        contents  => $yui3_all_css },
+        "robots.txt"              => { perms     => $CGI_READ,
+                                       overwrite => 1,
+                                       contents  => \&robots_txt},
     );
 
+    sub robots_txt {
+    my $output = '';
+    Bugzilla->template->process("global/robots.txt.html", {}, \$output);
```

Comment:
In the git move, you put the robots.txt into template/en/default/robots.txt.tmpl. You need to move it to .../global/robots.txt.tmpl.